api/tests/integration/migrations/utils.js

```diff
@@ -192,191 +175,88 @@ const switchToRealDbs = () => {
 };
 
 const switchToTestDbs = () => {
-  db.audit = db.use(DB_PREFIX + 'audit');
-  db.medic = db.use(DB_PREFIX + 'medic');
+  db.medic = db.use('medic-test');
```

Comment:
Is it OK that this uses the same DB that E2E uses? I guess they don't run at the same time..

Reply:
Yeah I think it's ok. There could be issues if you ran them both locally, but they both do setup and teardown so even that should be alright. Just don't run them at the same time!

The benefit is it simplifies the setup code in CI because we don't need all the switches.